README.md

```diff
@@ -87,7 +87,7 @@ Unikraft is licensed under a BSD-3-Clause.  For more information, please refer t
 [unikraft-website]: http://unikraft.org
 [unikraft-docs]: http://docs.unikraft.org
 [unikraft-guides]: https://github.com/unikraft/unikraft/tree/staging/docs/guides
-[unikraft-contributing]: https://github.com/unikraft/unikraft/blob/staging/CONTRIBUTING.md
+[unikraft-contributing]: https://unikraft.org/docs/contributing/
 [unikraft-ci]: http://ci.unikraft.org
```

Comment:
This link is broken but I don't know what should be here.

Reply:
Link works my side :) Maybe it didn't then